setup.py

```diff
@@ -153,19 +208,19 @@ def make_absolute_imports(self, compiled_files):
                 f.truncate()
 
 
-class build(build.build, BuildGRPC):
+class BuildProtos(build.build, BuildGRPC):
     def run(self, *args, **kwargs):
```

Comment:
We should remove the args here too if its a no-op. Same with the others

Reply:
Same as below.